src/index.js

```diff
@@ -169,7 +172,7 @@ export default function renderToString(vnode, context, opts, inner) {
 		for (let i=0; i<len; i++) {
 			let child = children[i];
 			if (!falsey(child)) {
-				let ret = renderToString(child, context, opts, true);
+				let ret = renderToString(child, context, opts, true, nodeName==='svg');
```

Comment:
Oh, good catch!

I want to capture this in a test but because the `render` doesn't actually use `isSvgMode` except for handling the `xlink` attributes, the only way I can think of is to assert that `<svg><foreignObject xlinkFoo /></svg>` renders the attribute as `xlinkFoo` and not `xlink:foo`. This feels a bit gross to me; what do you think?